pkg/printers/internalversion/describe.go

```diff
@@ -2478,6 +2478,26 @@ func describeNodeResource(nodeNonTerminatedPodsList *api.PodList, node *api.Node
 	w.Write(LEVEL_1, "%s (%d%%)\t%s (%d%%)\t%s (%d%%)\t%s (%d%%)\n",
 		cpuReqs.String(), int64(fractionCpuReqs), cpuLimits.String(), int64(fractionCpuLimits),
 		memoryReqs.String(), int64(fractionMemoryReqs), memoryLimits.String(), int64(fractionMemoryLimits))
+
+	oirResource := make([]string, 0, len(allocatable))
+	for resource := range allocatable {
+		if helper.IsOpaqueIntResourceName(resource) {
+			oirResource = append(oirResource, string(resource))
+		}
+	}
+	sort.Strings(oirResource)
+	for _, oir := range oirResource {
+		shortOirName := strings.Replace(oir, api.ResourceOpaqueIntPrefix, "OIR-", -1)
```

Comment:
You may want to retain this, since OIRs will be included in the set of extended resources for the time being.